### ISSUE TITLE
PhasePlot does not support annotate_line

### ISSUE THREAD

**Issue by yt-fido** [yt-project/yt#798]
Originally reported by: **Stephanie Tonnesen (Bitbucket: [stonnes](https://bitbucket.org/stonnes), GitHub: [stonnes](https://github.com/stonnes))**

----------------------------------------

AttributeError:  'PhasePlot' object has no attribute 'annotate_line' in Version 2.7-dev that I updated March 5. 

----------------------------------------
- Bitbucket: https://bitbucket.org/yt_analysis/yt/issue/798

**Comment by henry2004y** [comment]
Hi! Is there any progress on adding this feature?

**Comment by neutrinoceros** [comment]
No. If you're interested in contributing a patch, it'd be very much welcome !
